packages/std/src/addresses.rs

```diff
@@ -52,6 +52,11 @@ impl Addr {
     pub fn as_str(&self) -> &str {
         self.0.as_str()
     }
+
+    #[inline]
+    pub fn as_bytes(&self) -> &[u8] {
```

Comment:
Could you add a bit of doc comment for all the function that are new or reworked? In this two things are interesting:
- This is a UTF-8 encoding of the address string
- It is equivalent to `addr.as_str().as_bytes()`

Reply:
Done!